packages/flutter_tools/lib/src/ios/ios_workflow.dart

```diff
@@ -90,6 +93,17 @@ class IOSWorkflow extends DoctorValidator implements Workflow {
       ));
     }
 
+    // Python dependencies installed
+    if (hasPythonSixModule) {
+      pythonStatus = ValidationType.installed;
+    } else {
+      pythonStatus = ValidationType.missing;
+      messages.add(new ValidationMessage.error(
+        'Python installation missing module "six".\n'
+        'Install via \'sudo easy_install six\'.'
```

Comment:
SGTM. I'll tweak the message. Another alternative is to mention that they may be using a custom install and suggest that they can fix this by using the default system version (without being to pushy about it).